core/variables.ts

```diff
@@ -202,7 +202,6 @@ export function generateUniqueNameFromOptions(
   let letterIndex = letters.indexOf(startChar);
   let potName = startChar;
 
-  // eslint-disable-next-line no-constant-condition
```

Comment:
This rule is still part of recommended. Does removing this line mean that you disabled no-constant-condition for all of Blockly, or that the recommended ruleset is no longer fully being applied?

Reply:
Per https://eslint.org/docs/latest/rules/no-constant-condition#checkloops, the `allExceptWhileTrue` option for this rule is also on by default, so this no longer triggers.